src/Enhavo/Bundle/AppBundle/Controller/ResourceController.php

```diff
@@ -230,4 +232,10 @@ public function deleteAction(Request $request)
         $this->domainManager->delete($this->findOr404($request));
         return new Response();
     }
-}
\ No newline at end of file
+
+    protected function dispatchEvent($eventName)
+    {
+        $dispatcher = $this->get('event_dispatcher');
+        $dispatcher->dispatch($eventName);
```

Comment:
Can add please an Event, so the listener is able to check what is the resource. You can use GenericEvent for example.